tests/acceptance/simple-test.js

```diff
@@ -0,0 +1,23 @@
+import Ember from 'ember';
+import { module, test } from 'qunit';
+import startApp from '../helpers/start-app';
+
+var App;
+
+module('Simple Acceptance Test', {
+  beforeEach: function() {
+    App = startApp();
+  },
+
+  afterEach: function() {
+    Ember.run(App, 'destroy');
+  }
+});
+
+test('value of input', function(assert) {
+  visit('/');
+
+  andThen(function() {
```

Comment:
also adding some es6 to the test tree would be great

`beforeEach() { }`
` andThen( () => {  }`